extensions/amp-video-docking/0.1/amp-video-docking.js

```diff
@@ -487,7 +502,7 @@ export class VideoDocking {
     }
 
     const scrollDirection =
```

Comment:
Is this `const` needed?

Reply:
yeah, it's assigned as member read later.